### PR TITLE
Add approvers to facilitate project management

### PR DESCRIPTION
**Pull request by brahmaroutu**
We would like to pick up pace in merging initial code to move to alpha implementation ASAP. This requires us to have some approvers who are involved deeply in this project. Please approve this PR ASAP to get help us carry the work at a faster pace.

/assign @xing-yang @saad-ali

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -5,3 +5,7 @@ approvers:
   - msau42
   - saad-ali
   - xing-yang
+  - wlan0
+  - brahmaroutu
+  - rrati
+
```
Comment (xing-yang) [inline OWNERS:11]: nit: remove the empty line

Comment (brahmaroutu) [reply OWNERS]: Fixed. Please review.

**Pull request merged**